### PR TITLE
Detective Vendor + Aerostatic Suits (Atomized)

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Ports Detective Vendor on all maps, Aerostatic Clothes unlocked by hacking the vendor, and changed detective loadout.

## Why It's Good For The Game

Atomized because wooden wardrobe is clustered.

Added holster to detective's roundstart gear so that the detective wardrobe cannot be tided and the stuff get stolen.

## Changelog
:cl:
add: Ported Detective Vendor Wardrobe on all maps
add: Ported Aerostatic Clothes, avalaible at Det Vendor
add: Detective now starts with a holster around his belt (and holsters can be worn in belt slot)
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -256,3 +256,9 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 	thermal_overload()
+
+/obj/item/clothing/glasses/hud/security/sunglasses/disco
+	name = "binoclard lenses"
+	desc = "Stylish round lenses subtly shaded for your protection and criminal discomfort. Has enhanced shielding which blocks flashes a security HUD."
```
Comment (yyzsong) [inline code/modules/clothing/glasses/hud.dm]: Missing an And in there

**Review by yyzsong — CHANGES_REQUESTED** [review]

**Comment by bloons3** [comment]
> Added holster to detective's roundstart gear so that the detective wardrobe cannot be tided and the stuff get stolen.

Makes it impossible for anyone promoted in round to actually assume the position though.

File: code/modules/vending/security.dm [hunk 2]
```diff
@@ -39,3 +39,68 @@
...
+					/obj/item/clothing/glasses/hud/security/sunglasses/disco = 1,
+					/obj/item/clothing/shoes/sneakers/disco = 1,
+					/obj/item/clothing/suit/det_suit/disco = 1,
+					/obj/item/clothing/under/rank/security/detective/disco = 1)
+	premium = list(	/obj/item/detective_scanner = 1,
+					/obj/item/twohanded/binoculars = 1,
+					/obj/item/camera/detective = 1,
+					/obj/item/holosign_creator/security = 1)
+
+	refill_canister = /obj/item/vending_refill/detective
```
Comment (bloons3) [inline code/modules/vending/security.dm:86]: does this refill actually exist in cargo?

File: code/modules/vending/security.dm [hunk 3]
```diff
@@ -39,3 +39,68 @@
...
+	extra_price = 100
```
Comment (bloons3) [inline code/modules/vending/security.dm:87]: Any attempt to actually balance this price?

**Comment by eeSPee** [comment]
> > Added holster to detective's roundstart gear so that the detective wardrobe cannot be tided and the stuff get stolen.
> 
> Makes it impossible for anyone promoted in round to actually assume the position though.

But then that raises further issues:
* any assistant that finds the money room can tide into det's office and get the gun
* you can buy refills and, therefore, multiple guns

File: code/modules/vending/security.dm (hunk 3 above)
Comment (eeSPee) [reply code/modules/vending/security.dm:87]: What do you suggest?

**Comment by bloons3** [comment]
> * you can buy refills and, therefore, multiple guns

You can already buy guns from Cargo. Also, if the HR event fires, it's possible to get multiple Detectives, who will each have their own gun.

**Comment by eeSPee** [comment]
Ok

**Comment by heepox** [comment]
some day

**Comment by eeSPee** [comment]
It's not a commented out feature people like, so naturally it will become Stale.

**Comment by ike709** [comment]
@eeSPee Post an in-game pic of the outfit(s). If they don't look terrible I'll merge when conflicts are fixed.

**Comment by eeSPee** [comment]
image

They're inspired by the game Disco Elysium.

image

**Pull request closed without merging**